module1/homework/make_unique/tests/tests.cpp

```diff
@@ -12,23 +13,68 @@ class makeUniqueTests : public ::testing::Test
    protected:
     int defaultValue{5};
     double anotherValue{10.5};
+    std::array<int, 5> defaultArray{1, 2, 3, 4, 5};
 };
 
-TEST_F(makeUniqueTests, noParametersShouldBeDefaultObject) {
+class makeUniqueTestsWithStdOut : public makeUniqueTests
+{
+    std::streambuf* coutBuffer{};
+
+   protected:
+    std::stringstream testStream{};
+    void SetUp() override
+    {
+        coutBuffer = std::cout.rdbuf();
+        std::cout.rdbuf(testStream.rdbuf());
+    }
+
+    void TearDown() override
+    {
+        std::cout.rdbuf(coutBuffer);
+        testStream.str(std::string());
+    }
+};
```

Comment:
we want to avoid testing things with stdin/stdout. In this case it can be avoided.